faux-mgs/src/main.rs

```diff
@@ -1471,6 +1585,58 @@ async fn monorail_unlock(
     Ok(())
 }
 
+fn ssh_keygen_sign(
+    pub_key_path: &Path,
+    data: &[u8],
+) -> Result<ssh_key::SshSig> {
+    use std::process::{Command, Stdio};
+    let mut child = Command::new("ssh-keygen")
```

Comment:
Use a deterministic path to the executable. See previous comment.

Reply:
Also fixed in e006752c131e !